build.gradle

```diff
@@ -88,6 +88,16 @@ task travisRelease {
     }
 }
 
+task installGitPreCommitHook(type: Copy) {
+    from 'pre-commit'
+    into '.git/hooks'
+}
+
+task makeGitHooksExecutable { Task t ->
+    t.dependsOn(installGitPreCommitHook)
+    Runtime.getRuntime().exec("chmod -R +x .git/hooks/");
```

Comment:
You're missing 'doLast' action here! Without it, the 'exec' is triggered during Gradle's 'configuration time', that is every time Gradle prepares for running any task :)

- [ ] - I suggest to use ExecTask for "makeGitHooksExecutable". See Gradle docs on ExecTask.